node/as/thrift.js

```diff
@@ -53,8 +55,29 @@ function TChannelAsThrift(opts) {
     self.logParseFailures = typeof logParseFailures === 'boolean' ?
         logParseFailures : true;
 
-    // only used in request()
     self.channel = opts.channel;
+
+    self.healthCheckCallback = opts.healthCheckCallback;
```

Comment:
Let's call this `self.isHealthy = opts.isHealthy`. This is not a callback so cant be called Callback.